server/index.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path');
+const express = require('express');
+const models = require('./models/api.js');
+
+const PORT = 2373;
+const app = express();
+
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+app.use(express.static(path.join('/../public')));
+
+app.get('/api/reviews/:id', (req, res) => {
+  const requestId = req.originalUrl.substring('/api/reviews/'.length);
```

Comment:
@Matt-Wigg there's actually a simpler way to do this.
req.params.id will give you the value of :id.

Reply:
Awesome! Thanks